nextflow.config

```diff
@@ -0,0 +1,122 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    icgc-argo-workflows/prealnqc Nextflow config file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Default config options for all compute environments
+----------------------------------------------------------------------------------------
+*/
+
+// Global default params, used in configs
+params {
+
+    // Input options
+    input                      = null
+    local_mode                 = false
+    study_id                   = null
+    analysis_ids               = null
+    cleanup                    = true
+
+    // rdpc data staging 
+    tempdir                    = null
+    api_token                  = null
+    song_url                   = null
+    score_url                  = null
+    song_url_download          = null
+    score_url_download         = null
+    api_download_token         = null
+    song_url_upload            = null
+    score_url_upload           = null
+    api_upload_token           = null
+    transport_parallel         = null
+    transport_mem              = null
+    song_container             = "ghcr.io/overture-stack/song-client"
+    song_container_version     = "5.0.2"
+    score_container            = "ghcr.io/overture-stack/score"
+    score_container_version    = "5.10.0"
+
+    // Boilerplate options
+    outdir                     = null
+    tracedir                   = params.outdir ? "${params.outdir}/pipeline_info" : null
+    publish_dir_mode           = 'copy'
+    email                      = null
+    email_on_fail              = null
+    plaintext_email            = false
+    monochrome_logs            = false
+    hook_url                   = null
+    help                       = false
+    version                    = false
+    validate_params            = true
+    show_hidden_params         = false
+    schema_ignore_params       = 'genomes'
+
+    // Config options
+    config_profile_description = null
+    config_profile_name        = null
+    multiqc_config             = null
+    multiqc_logo               = null
+
+
+}
+
```

Comment:
You can add the similar portion to let the nextflow auto-download the files from amazon s3 bucket:
https://github.com/icgc-argo-workflows/dnaalnqc/blob/main/nextflow.config#L90-L103